src/Core/Framework/DataAbstractionLayer/ExtensionRegistry.php

```diff
@@ -27,4 +32,50 @@ public function getExtensions(): iterable
     {
         return $this->extensions;
     }
+
+    /**
+     * @return array<EntityExtension>
+     */
+    public function buildBulkExtensions(DefinitionInstanceRegistry $registry): array
+    {
+        $extensions = [];
+
+        foreach ($this->bulks as $bulk) {
+            foreach ($bulk->collect() as $entity => $fields) {
+                try {
+                    $definition = $registry->getByEntityName($entity);
```

Comment:
we can mark this as deprecated as with relying on entity names we don't need to fetch the definition here right?